docs/user-defined-redirects.rst

```diff
@@ -39,11 +40,19 @@ The example configuration would be::
 
 Your users query would now redirect in the following manner::
 
-	docs.example.com/dev/install.html ->
-	docs.example.com/en/latest/install.html
+        docs.example.com/dev/install.html ->
+        docs.example.com/en/latest/install.html
 
 Where ``en`` and ``latest`` are the default language and version values for your project.
 
+
+.. note::
+
+   In other words, a *Prefix Redirect* removes a prefix from the original URL.
```

Comment:
Not sure if this note is necessary, I found it a little confusing, the above example makes more sense for me

Reply:
I want to have a simple way to explain what `Prefix Redirects` does because it seems that explaining it by example with a "custom domain" is not enough and confuses people that don't use a custom domain.

So, I'd like to have something very simple explained in one line. Maybe my one-line explanation is not the best and someone can help me here. But I'd like to keep a note like this.